src/main/java/uk/gov/hmcts/cmc/claimstore/utils/PartyNameUtils.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.cmc.claimstore.utils;
+
+import uk.gov.hmcts.cmc.domain.models.party.NamedParty;
+import uk.gov.hmcts.cmc.domain.models.party.SoleTrader;
+
+public class PartyNameUtils {
+
+    private PartyNameUtils() {
+        // Utility class, no instances
+    }
+
+    public static String getPartyNameFor(NamedParty party) {
+        StringBuilder name = new StringBuilder(party.getName());
+
+        if (party instanceof SoleTrader) {
+            (((SoleTrader) party).getBusinessName()).ifPresent(t -> name.append(" T/A ").append(t));
```

Comment:
is this business requirement?

Reply:
not in this story, but that's what we already had for other letters addressed to the claimant. Will check with Sanchita